packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/index.hbs

```diff
@@ -28,22 +28,23 @@
             @icon="spend"
             @sign="§"
             @symbol="SPEND"
-            @balance={{this.selectedFaceValue}}
-            @usdBalance={{spend-to-usd this.selectedFaceValue}}
-            @text={{concat "≈ " (spend-to-usd this.selectedFaceValue) " " this.fundingTokenSymbol "*"}}
+            @balance={{this.selectedFaceValue.spendAmount}}
+            @usdBalance={{spend-to-usd this.selectedFaceValue.spendAmount}}
+            @text={{concat this.selectedFaceValue.tokenAmount " " this.fundingTokenSymbol "*"}}
             data-test-face-value-display
           />
         </CardPay::LabeledValue>
       {{else}}
         <div class="face-value-card__options">
-          {{#each this.faceValueOptions as |amount|}}
+          {{#each this.options as |option|}}
             <CardPay::IssuePrepaidCardWorkflow::FaceValue::Option
-              @disabled={{lt this.balanceFromBN (spend-to-usd amount)}}
-              @checked={{eq amount this.selectedFaceValue}}
-              @spendFaceValue={{amount}}
-              @usdAmount={{spend-to-usd amount}}
-              @approxDaiAmount={{spend-to-usd amount}}
-              @onInput={{fn this.chooseFaceValue amount}}
+              @disabled={{lt this.balanceFromBN (spend-to-usd option.spendAmount)}}
```

Comment:
Just realised that `balanceFromBN` is not accurate conversion of the token balance to USD. If you want to compare in USD, I think using the `token-to-usd` helper or service and converting to a number is better.

Comparing USD might not be good because we are losing decimal precision. It might be better to compare the raw token amounts fetched in `getTokenAmounts` vs the balance as `BN`.